protodocparser_test.go

```diff
@@ -61,14 +97,14 @@ func TestServiceNames(t *testing.T) {
 	}
 }
 
-func TestParseSimpleProto(t *testing.T) {
+func xTestParseSimpleProto(t *testing.T) {
```

Comment:
Yep, I disabled the test, since it is broken but noisy:

[] rebase and make i trun